lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -370,37 +348,6 @@ def test_no_certificate(self):
         certificate_data = response.data[0]['certificate']
         self.assertDictEqual(certificate_data, {})
 
-    @patch.dict(settings.FEATURES, {'CERTIFICATES_HTML_VIEW': False, 'ENABLE_MKTG_SITE': True})
-    def test_pdf_certificate_with_html_cert_disabled(self):
-        """
-        Tests PDF certificates with CERTIFICATES_HTML_VIEW set to True.
-        """
-        self.verify_pdf_certificate()
-
-    @patch.dict(settings.FEATURES, {'CERTIFICATES_HTML_VIEW': True, 'ENABLE_MKTG_SITE': True})
-    def test_pdf_certificate_with_html_cert_enabled(self):
-        """
-        Tests PDF certificates with CERTIFICATES_HTML_VIEW set to True.
-        """
-        self.verify_pdf_certificate()
-
-    @patch.dict(settings.FEATURES, {'CERTIFICATES_HTML_VIEW': True, 'ENABLE_MKTG_SITE': True})
-    def test_web_certificate(self):
```

Comment:
Why do you deleting this test? Looks like it cheks the certificate web view URL, not the download URL

Reply:
This is the test where i left empty url in `serializer.py`